README.md

```diff
@@ -257,3 +299,20 @@ Destination is relative to the output policy set (`out/masterfiles`).
 - Enable `services_autorun_bundles` class in `def.json`.
 - Merge any `def.json` recursively into `out/masterfiles/def.json`.
 - Copy any other files with their existing directory structure to destination.
+
+#### `bundles <bundles ...>`
+- Ensure bundles are evaluated by adding them to the bundle sequence, using `def.json`.
+- Only manipulates the bundle sequence, to ensure policy files are copied and parsed, use other build steps, for example `copy` and `policy_files`.
+
+#### `policy_files <paths ...>`
+- Add policy (`.cf`) files to `inputs` key in `def.json`, ensuring they are parsed.
+  - Only edits `def.json`, does not copy files. Should be used after a `copy` or `directory` build step.
+  - Does not add any bundles to the bundle sequence, to ensure a bundle is evaluated, use the `bundles` build step or the autorun mechanism.
+- All paths are relative to `out/masterfiles`.
+- If any of the paths are directories (end with `/`), the folder(s) are searched and all `.cf` files are added.
```

Comment:
searched recursively?

Reply:
If you use a directory as argument instead of a policy file; it will search through the directory and its sub-directories for any files that with the `.cf` file extension and add them to inputs.